src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/Helpers/SafeNativeMethods.vb

```diff
@@ -0,0 +1,34 @@
+' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Security
+Imports System.Runtime.InteropServices
+Imports System.Runtime.Versioning
+
+Namespace Microsoft.VisualBasic.CompilerServices
+
+    <System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)> _
+    <SecurityCritical()> _
```

Comment:
SecurityCritical, SuppressUnmanagedCodeSecurityAttribute, ResourceExposure are all attributes that don't have function in .NET Core.